src/pages/workspace/expensifyCard/WorkspaceExpensifyCardPage.tsx

```diff
@@ -0,0 +1,172 @@
+
+    const policyCurrency = useMemo(() => policy?.outputCurrency ?? CONST.CURRENCY.USD, [policy]);
+
+    // TODO: uncomment the code line below to use cardsList data from Onyx when it's supported
+    // const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARDS_LIST}${policyID}_Expensify Card`);
+    const cardsList = mockedCards;
+
+    const fetchExpensifyCards = useCallback(() => {
+        // TODO: uncomment when OpenPolicyExpensifyCardsPage API call is supported
+        // Policy.openPolicyExpensifyCardsPage(policyID);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [policyID]);
+
+    useFocusEffect(fetchExpensifyCards);
+
+    const sortedCards = useMemo(
+        () =>
+            Object.values(cardsList ?? {}).sort((a, b) => {
+                // TODO: change cardholder to accountID and get personal details with it
+                const aName = PersonalDetailsUtils.getDisplayNameOrDefault(a.cardholder);
+                const bName = PersonalDetailsUtils.getDisplayNameOrDefault(b.cardholder);
+                return localeCompare(aName, bName);
+            }),
+        [cardsList],
+    );
+
+    const getHeaderButtons = () => (
+        <View style={[styles.w100, styles.flexRow, styles.gap2, shouldUseNarrowLayout && styles.mb3]}>
+            <Button
+                medium
+                success
+                onPress={() => {}} // TODO: add navigation action when card issue flow is implemented (https://github.com/Expensify/App/issues/44309)
+                icon={Expensicons.Plus}
+                text={translate('workspace.expensifyCard.issueCard')}
+                style={shouldUseNarrowLayout && styles.flex1}
+            />
+            <Button
+                medium
+                onPress={() => {}} // TODO: add navigation action when settings screen is implemented (https://github.com/Expensify/App/issues/44311)
+                icon={Expensicons.Gear}
+                text={translate('common.settings')}
+                style={shouldUseNarrowLayout && styles.flex1}
+            />
+        </View>
+    );
+
+    const renderItem = ({item, index}: ListRenderItemInfo<Card>) => (
+        <OfflineWithFeedback
+            key={`${item.name}_${index}`}
+            pendingAction={item.pendingAction}
+            errorRowStyles={styles.ph5}
+            errors={item.errors}
+        >
+            <PressableWithoutFeedback
+                role={CONST.ROLE.BUTTON}
+                accessibilityLabel="row"
+                onPress={() => {}} // TODO: add navigation action when card details screen is implemented (https://github.com/Expensify/App/issues/44325)
+            >
+                {({hovered}) => (
+                    <WorkspaceCardListRow
+                        style={hovered && styles.hoveredComponentBG}
+                        lastFourPAN={item.lastFourPAN}
+                        cardholder={item.cardholder}
+                        limit={item.limit}
```

Comment:
The limit will be in card.nameValuePairs.unapprovedExpenseLimit

https://github.com/Expensify/App/blob/3622deb409a4cf91a47cc9196e2414ffe75f0f16/src/types/onyx/Card.ts#L51-L55